backend/src/nodes/impl/pytorch/convert_to_onnx_impl.py

```diff
@@ -0,0 +1,42 @@
+from io import BytesIO
+
+import torch
+from spandrel import ModelDescriptor
+
+
+def convert_to_onnx_impl(
+    model: ModelDescriptor,
+    device: torch.device,
+    use_half: bool = False,
+    input_name: str = "input",
+    output_name: str = "output",
+) -> bytes:
+    # https://github.com/onnx/onnx/issues/654
+    dynamic_axes = {
+        input_name: {0: "batch_size", 2: "height", 3: "width"},
+        output_name: {0: "batch_size", 2: "height", 3: "width"},
+    }
+    dummy_input = torch.rand(1, model.input_channels, 64, 64)
```

Comment:
64 as to be a valid input size for the model, right? If so, then we might want to round this up to the nearest multiple of `model.size_requirements.multiple_of`.